addon/components/plugins/html-editor/modal.ts

```diff
@@ -0,0 +1,52 @@
+import Component from '@glimmer/component';
+import { localCopy } from 'tracked-toolbox';
+import beautify from 'js-beautify';
+import { action } from '@ember/object';
+import { basicSetup } from 'codemirror';
+import { html } from '@codemirror/lang-html';
+import CodeMirrorModifier from '@lblod/ember-rdfa-editor/modifiers/_private/code-mirror';
+
+type Args = {
+  content: string;
+  onSave: (content: string) => void;
+  onCancel: () => void;
+};
+
+export default class HTMLEditorModal extends Component<Args> {
+  CodeMirror = CodeMirrorModifier;
+
+  @localCopy('args.content') declare content: string;
+
+  get formattedContent() {
+    return beautify.html(this.content, {
```

Comment:
Looking at some examples in the editor, I find it easier to read (though a little aggressively spread onto new lines) with e.g. the ``wrap_attributes:` 'force-expand-multiline'` option too.

Reply:
Good point, I'll also look into the other possible values of `wrap_attributes`